### DISCUSSION TITLE
Dragging tabs to change tab order

### DISCUSSION THREAD

**Discussion by naxey** [roovo/obsidian-card-board#172] [Ideas]
<img width="676" alt="kanban tabs" src="https://github.com/roovo/obsidian-card-board/assets/40920984/652dd2fe-0e95-491e-ba02-bed623c5849c">

**Comment by roovo** [comment]
Hi @Naxey -> this is now possible - or alternatively you can drag the board names in the settings pannel to do the same thing.
